### ISSUE TITLE
Redirecting back to home instance to execute a remote follow gives no visual indication anything is happening

### ISSUE THREAD

**Issue by graytFox** [mastodon/mastodon#18147]
### Steps to reproduce the problem

1. Sign up on instance A
2. Go to the user directory of instance B and click the follow button
3. On the pop-up window that appears asking for your home instance, enter your instance domain
4. Click "Proceed to follow" button

### Expected behaviour

The page should give some indication that it's performing work (presumably looking up the instance), such as disabling the "Proceed to follow" button or showing a loading spinner

### Actual behaviour

Nothing changes on the page until it's finished looking up the instance and starts redirecting the user to their home server

### Specifications

3.4.6, first observed on botsin.space. Observed with Google Chrome on Linux.

**Comment by ClearlyClaire** [comment]
Agreed, between the server-side request and the redirection, it can take significantly longer for the user to get anything back than on other pages. I'm not sure what the UI should look like though, I'm not sure that disabling the “Proceed to follow” button would be more understandable (a spinner would probably work though).
